package.json

```
@@ -5,7 +5,7 @@
   "main": "dist/index.js",
   "scripts": {
     "build": "yarn lint && ncc build index.js",
-    "lint": "eslint index.js"
+    "lint": "eslint index.js --fix"
```

Please keep the default lint script without fix (to be used for CI later):

```suggestion
    "lint": "eslint index.js",
    "lint:fix": "eslint index.js --fix"
```